website_sale_product_attribute_value_filter_existing/static/src/js/website_sale_product_attribute_value_filter_existing_tour.esm.js

```diff
@@ -8,26 +8,33 @@ tour.register(
         test: true,
         url: "/shop",
     },
+    // No product has the yellow colour attribute defined.
+    // When enter to "/shop" the attribute "test yellow" should not appear in the list of filters by attribute.
     [
+        // For the following steps it is checked that the attribute "test yellow" is
+        // not present in the list but the attributes "test red", "test blue" and
+        // "test green" must be present.
```

Comment:
Good, but synthesize it to put it in the `content` property so if the test fails it's obvious what was expected from the logs

Reply:
Done, thanks!